### DISCUSSION TITLE
Ability to apply a CSS class or style to a specific table row

### DISCUSSION THREAD

**Discussion by ziming** [filamentphp/filament#2746] [Ideas]
It will be nice we can "feature" a particular row in filament table with CSS style or CSS classes.

Makes for some use case such as bringing attention to a particular record more so than others.

**Comment by simonbuehler** [comment]
sprinkling in some additions css classes like `even`, `odd`, would be great for styling

**Reply by danharrin** [reply]
Feel free to PR these 👍

**Comment by danharrin** [comment]
#3485
